packages/palette/src/elements/Shelf/Shelf.tsx

```diff
@@ -222,9 +217,7 @@ export const Shelf: React.FC<ShelfProps> = ({
         </Viewport>
       </FullBleed>
 
-      {showProgress && (
-        <CarouselBar mt={6} transition="none" percentComplete={progress} />
-      )}
+      {showProgress && <ShelfScrollBar viewport={viewportRef.current} />}
```

Comment:
Can we think of any instances where we'd want to disable scrollbar dragging? On mobile or something?

Reply:
It won't work on mobile (we *could* implement it but it's kind of an unnatural UI pattern for touch since you've got inertial scroll right there). The tap to scroll does work however.